Adding AXE & Puppeteer Example for HTML code (#1)
nkelkar-kg/Axe-Puppeteer-and-HTML: README.md

# Axe-Puppeteer-and-HTML
This repo is an example of using Puppeteer to run accessibility checks on HTML code. The main script that is run is called `a11y-check.js`

> Essentially, Puppeteer runs in headless mode, and the axe-core package code is injected into the "browser" as script. If axe finds any serious or critical issues, it will output the array of violations into the console.

Reviewer: ```suggestion
Puppeteer is a browser automation tool that can be run in headless mode, we can then inject the axe-core package into the headless browser as a script. If Axe finds any serious or critical issues, it will output the array of violations into the console.
```

> `process.exit(1)` is used since if one wanted to add this script in a husky pre-push, any accessibility issues would prevent the developer from pushing their code to their remote repo.

Reviewer: ```suggestion
We use `process.exit(1)` to make this script fail in checks. For example, in a husky git hook on pre-push, any accessibility issues would prevent the developer from pushing their code to their remote repo.
```
